AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/participant.model.ts

```diff
@@ -46,4 +46,8 @@ export class ParticipantModel {
               }
             : null;
     }
+
+    static IsEmailEjud(email: string): boolean {
+        return email && email.toLowerCase().includes('judiciary');
```

Comment:
I'm just considering is that better if we can use regex validate Ejud email here?

Reply:
Not sure if that makes a difference, Regex for pattern match & validation, This code is to identify if email belongs to AAD judge or ejud. 
server side code is also on similar lines in HearingDetailsResponseExtensions
private static bool IsEmailEjud(string email)
{
    return !string.IsNullOrEmpty(email)  && email.Contains("judiciary", StringComparison.CurrentCultureIgnoreCase);
}